.github/set-urls-for-review.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -e
+
+setUrlsToDomainsUrls () {
+    DOMAINS=${HOSTS}
+    ITERATOR=1
+
+    for DOMAIN in ${DOMAINS//,/ } ; do
+        docker compose exec -T php-fpm sed -i "s/http:\/\/127.0.0.${ITERATOR}:8000/https:\/\/${DOMAIN}/g" project-base/config/domains_urls.yaml
```

Comment:
maybe it could be rewritten without docker compose as a script that is meant to be run inside the container. 
Then it would not be necessary to transfer the file through artifacts

Reply:
It is dockerignored, that is why I need to move it